crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
+        }
+        // typo in println() function
+        prentln("Sum: $sum")
+        // Unnecessary object creation inside a loop
+        for (i in 0 until 100000) {
+            val tempObject = Any() // Creates a new object on each iteration
+        }
+        // Inefficient string concatenation
+        var longString = ""
```

Comment:
Avoid blocking the main thread with `Thread.sleep`; consider using asynchronous processing.
```suggestion
// Inefficient way to calculate the sum of elements
    var sum = 0
    fun calculateSum() {
        sum = largeList.sum()
        // typo in println() function
        println("Sum: $sum")
        // Unnecessary object creation inside a loop
        val tempObject = Any() // Create the object once outside the loop
        for (i in 0 until 100000) {
            // Use tempObject if needed
        }
        // Inefficient string concatenation
        var longString = ""
```